Makefile

```diff
@@ -4,7 +4,8 @@ IMAGE_REPO ?= nmstate
 HANDLER_IMAGE_NAME ?= kubernetes-nmstate-handler
 HANDLER_IMAGE_SUFFIX ?=
 HANDLER_IMAGE_FULL_NAME ?= $(IMAGE_REPO)/$(HANDLER_IMAGE_NAME)$(HANDLER_IMAGE_SUFFIX)
-HANDLER_IMAGE ?= $(IMAGE_REGISTRY)/$(HANDLER_IMAGE_FULL_NAME)
+IMAGE_TAG ?= "latest"
```

Comment:
Could we explicitly pass latest from the Travis config for master? This makefile is do long it is starting to bother me.

Reply:
I am not sure if we can pass that we can just use the latest in the `HANDLER_IMAGE_SUFFIX`